lang/java/reef-io/src/main/java/org/apache/reef/io/data/loading/api/DataLoader.java

```diff
@@ -86,7 +99,41 @@ public DataLoader(
       @Parameter(DataLoadingRequestBuilder.DataLoadingEvaluatorMemoryMB.class) final int dataEvalMemoryMB,
       @Parameter(DataLoadingRequestBuilder.DataLoadingEvaluatorNumberOfCores.class) final int dataEvalCore,
       @Parameter(DataLoadingRequestBuilder.DataLoadingComputeRequest.class) final String serializedComputeRequest) {
```

Comment:
The `serializedComputeRequest` will never be `null`. If Tang has a binding for it, it will have a value. If there is no binding for it, Tang won't call this constructor.